torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -909,6 +952,7 @@ def __init__(
         param_init_fn: Optional[Callable[[nn.Module], None]] = None,
         device_id: Optional[Union[int, torch.device]] = None,
         sync_module_states: bool = False,
+        limit_all_gathers: bool = True,
```

Comment:
default seems inconsistent with the doc?

Reply:
Good catch! This is a mistake.